extensions-contrib/compressed-bigdecimal/pom.xml

```diff
@@ -65,7 +65,7 @@
     <dependency>
       <groupId>org.apache.calcite</groupId>
       <artifactId>calcite-core</artifactId>
-      <version>1.21.0</version>
+      <version>1.32.0</version>
```

Comment:
I think we should use `${calcite.version}` , which is declared in the root pom.xml, instead of 1.21.0 here for consistency. Althoug this does not address the version bump suggested by dependentbot in this here. @rash67